std/algorithm/iteration.d

```diff
@@ -1319,7 +1319,7 @@ private struct FilterBidiResult(alias pred, Range)
     }
 }
 
-// group
+/// group
 struct Group(alias pred, R) if (isInputRange!R)
```

Comment:
Yeah, and you should move it to under the factory function. No one should be using the struct directly, but since it's public we can't make it a static struct without deprecation cycles that are of questional value anyway.